src/main/java/net/rubygrapefruit/platform/internal/jni/NativeLogger.java

```diff
@@ -9,13 +9,15 @@
     static final Logger LOGGER = Logger.getLogger(NativeLogger.class.getName());
 
     enum LogLevel {
+        ALL(Level.ALL),
```

Comment:
I think we need to update the `LogLevel` enum on the native side as well, right?

Reply:
You're right. Should be fixed.